src/System.Runtime.Serialization.Primitives/src/System.Runtime.Serialization.Primitives.csproj

```diff
@@ -4,15 +4,17 @@
   <PropertyGroup>
     <AssemblyName>System.Runtime.Serialization.Primitives</AssemblyName>
     <AssemblyVersion>4.2.0.0</AssemblyVersion>
```

Comment:
FYI - I've reverted some of the versioning changes to this library in master see PR https://github.com/dotnet/corefx/pull/10707. We will need to handle that merge accordingly.